extensions/oidc-db-token-state-manager/runtime/src/main/java/io/quarkus/oidc/db/token/state/manager/runtime/OidcDbTokenStateManager.java

```diff
@@ -0,0 +1,150 @@
+                                .execute(Tuple.of(tokenState))
+                                .toCompletionStage())
+                .onFailure().transform(new Function<Throwable, Throwable>() {
+                    @Override
+                    public Throwable apply(Throwable throwable) {
+                        return new AuthenticationCompletionException(FAILED_TO_ACQUIRE_TOKEN, throwable);
+                    }
+                })
+                .flatMap(new Function<RowSet<Row>, Uni<? extends AuthorizationCodeTokens>>() {
+                    @Override
+                    public Uni<? extends AuthorizationCodeTokens> apply(RowSet<Row> rows) {
+                        if (rows != null) {
+                            final RowIterator<Row> iterator = rows.iterator();
+                            if (iterator.hasNext()) {
+                                final Row firstRow = iterator.next();
+                                return Uni
+                                        .createFrom()
+                                        .item(new AuthorizationCodeTokens(
+                                                firstRow.getString("id_token"),
+                                                firstRow.getString("access_token"),
+                                                firstRow.getString("refresh_token")));
+                            }
+                        }
+                        return Uni.createFrom().failure(new AuthenticationCompletionException(FAILED_TO_ACQUIRE_TOKEN));
+                    }
+                })
+                .memoize().indefinitely();
+    }
+
+    @Override
+    public Uni<Void> deleteTokens(RoutingContext routingContext, OidcTenantConfig oidcConfig, String tokenState,
+            OidcRequestContext<Void> requestContext) {
+        return Uni
+                .createFrom()
+                .completionStage(pool
+                        .preparedQuery(deleteStatement)
+                        .execute(Tuple.of(tokenState))
+                        .toCompletionStage())
+                .replaceWithVoid()
+                .onFailure()
+                .recoverWithItem(new Function<Throwable, Void>() {
+                    @Override
+                    public Void apply(Throwable throwable) {
+                        LOG.debugf("Failed to delete tokens: %s", throwable.getMessage());
+                        return null;
+                    }
+                });
+    }
+
+    static long now() {
+        return Instant.now().getEpochSecond();
+    }
+
+    private static long expiresIn(RoutingContext event, OidcTenantConfig oidcConfig) {
+        Long maxAge = event.<Long> get(SESSION_MAX_AGE_PARAM);
+        if (maxAge == null) {
+            // illegal state that shouldn't happen
+            throw new AuthenticationFailedException(SESSION_MAX_AGE_PARAM + " parameter is missing");
+        }
+        maxAge += oidcConfig.token.lifespanGrace.orElse(0);
```

Comment:
`sesson_max_age` property already includes lifespan and session age extension, see https://github.com/quarkusio/quarkus/blob/main/extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java#L940C34-L940C34

Reply:
Will fix, thank you